cura/CuraApplication.py

```diff
@@ -1320,3 +1333,10 @@ def checkIsValidProjectFile(self, file_url):
         except Exception as e:
             Logger.log("e", "Could not check file %s: %s", file_url, e)
             return False
+
+    def _onContextMenuRequested(self, x, y):
```

Comment:
Type hinting please (also for protected functions, as it's also there to protect us from mistakes)